application/dash_application/pages.py

```diff
@@ -29,6 +34,17 @@
 
 # Please do not edit this loop
 for name in pagenames:
-    pages.append(importlib.import_module(prefix + name))
+    try:
+        pages.append(importlib.import_module(prefix + name))
+    except (IOError, OSError):
+        print(traceback.format_exc(), file=sys.stderr)
+        ErrorPage = namedtuple('ErrorPage', 'layout title dataversion page_name init_callbacks')
```

Comment:
This should probably go somewhere more top-level.

Reply:
Done